akka-docs/src/main/paradox/stream/operators/Source-or-Flow/conflate.md

```diff
@@ -18,6 +18,19 @@ Allow for a slower downstream by passing incoming elements and a summary into an
 there is backpressure. The summary value must be of the same type as the incoming elements, for example the sum or
 average of incoming numbers, if aggregation should lead to a different type `conflateWithSeed` can be used:
 
+## Example
+
+Scala
+:   @@snip [SourceOrFlow.scala](/akka-docs/src/test/scala/docs/stream/operators/SourceOrFlow.scala) { #conflate }
+
+Java
+:   @@snip [SourceOrFlow.java](/akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java) { #conflate }
+
+If downstream is slower, the elements would be conflated by summing them
+such that the upstream can continue producing elements, while waiting for downstream demand
+e.g. no demand for some time, and later demand would result in `111` being emitted later on.
```

Comment:
I think this could be simplified (and fix mixed tense). What about something like this: 
"If downstream is slower the elements is conflated by summing them. This means that upstream can continue producing elements while downstream is applying backpressure. For example: downstream is backpressuring while `1`, `10` and `100` arrives from upstream, then backpressure stops and the conflated `111` is emitted downstream."

Reply:
Thanks, applied